docs/source/configuration/telemetry/instrumentation/instruments.mdx

```diff
@@ -114,6 +114,14 @@ telemetry:
           type: counter
           unit: count
           description: "my description"
+
+      graphql:
+        acme.graphql.list.lengths:
+          value:
+            list_length: value
+          type: histogram
+          unit: count
+          description: "my description"
```

Comment:
Line 82 should be updated to describe what this graphql instrument does/how it compares to the other three custom instruments.

Reply:
[7669dd2](https://github.com/apollographql/router/pull/5472/commits/7669dd2b783acacfa3addf7c9664270f134138ce)